.gitignore

```diff
@@ -127,3 +127,4 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+.idea
```

Comment:
please add a [git global ignore](https://stackoverflow.com/questions/7335420/global-git-ignore) and add it there.

Reply:
thanks will update